rest/model/patch.go

```diff
@@ -73,6 +74,12 @@ type APIChildPatchAlias struct {
 	PatchID *string `json:"patch_id"`
 }
 
+type APIPatchTriggerAlias struct {
```

Comment:
Do we need to create an API type for something that is only being used in the UI? Mohamed would know better but I would think we could just use a graphql type instead.

Reply:
Good point, however I think I'll go ahead with combining the fields like you suggested above. Perhaps in that case it's appropriate to maintain `APIPatchTriggerDefinition` as an API type as you configured it in https://github.com/evergreen-ci/evergreen/pull/4995?